docs/user/interactive_customizing.rst

```diff
@@ -164,6 +174,58 @@ these base ``Dockerfile`` s and add the ``renku``, ``git``, and ``jupyter``
 parts to another base image that you might have.
 
 
+Renku project configurations
+----------------------------
+
+When starting a new Interactive Environment, most of the options can be manually
+changed by the user. Depending on the specific RenkuLab deployment, you can select
+more RAM, a higher CPU quota, etc.
+
+Your project may even include a package with an advanced UI (like
+`Streamlit <https://renku.discourse.group/t/how-to-deploy-streamlit-in-renku/169>`_)
+and you probably want to choose it as default.
+
+It's possible to set a default value for all these options using the project
+configurations stored in the ``.renku/renku.ini`` file.
+Once you do that, each time a user tries to start a new environment, those options will
+be pre-selected.
+
+.. note::
+
+  Manually modifying the ``renku.ini`` file is not recommended.
+  You can use the
+  `renku config command <https://renku-python.readthedocs.io/en/latest/commands.html#module-renku.cli.config>`_
+  form an interactive environment.
+
+    renku config set interactive.default_url "/tree"
+
+  We are working on adding a user friendly solution to set default options on
```

Comment:
```suggestion
  We are working on adding a user-friendly solution to set default options on
```

Reply:
:+1: 
I'll fix it also on the other locations where we use the space